### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Kawhi-L805**

### CODE REVIEW

File: calcDisc test.R [hunk 1]
```diff
@@ -0,0 +1,22 @@
+library(testthat)
+
+# Ensure this path points directly to where calcDisc.r is located
+library(comf)
```
Comment (TwinGan) [inline calcDisc test.R]: delete those two import

File: calcDisc test.R [hunk 2]
```diff
@@ -0,0 +1,22 @@
+library(testthat)
+
+# Ensure this path points directly to where calcDisc.r is located
+library(comf)
+
+test_that("test calcDisc function", {
+  # Test 1: Simple input
+  result1 <- calcDisc(ta=25, tr=25, vel=0.1, rh=50, clo=0.5, met=1)
```
Comment (TwinGan) [inline calcDisc test.R]: where did this data come from?

File: calcDisc test.R [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  
+  # Test 2: Edge case with higher temperature
+  result2 <- calcDisc(ta=35, tr=35, vel=0.1, rh=50, clo=0.5, met=1)
+  expect_true(result2 > result1, info = "Expected higher discomfort with higher temperature")
+  
+  # Test 3: Variation in humidity
+  result3_low_humidity <- calcDisc(ta=25, tr=25, vel=0.1, rh=10, clo=0.5, met=1)
+  result3_high_humidity <- calcDisc(ta=25, tr=25, vel=0.1, rh=90, clo=0.5, met=1)
+  expect_true(result3_high_humidity > result3_low_humidity, info = "Expected higher discomfort with higher humidity")
+
```
Comment (TwinGan) [inline calcDisc test.R:30]: This whole function calcDisc does not exist in Python. Did you design it by yourself?

**Review by TwinGan — CHANGES_REQUESTED** [review]
we need discussion to decide how to do this unit test

File: test_calcpmv_pdd.R [hunk 4]
```diff
@@ -0,0 +1,132 @@
+# Define helper functions for PMV and PPD calculations
+pmv_ppd <- function(tdb, tr, vr, rh, met, clo, standard = "ISO", units = "SI", limit_inputs = TRUE) {
```
Comment (TwinGan) [inline test_calcpmv_pdd.R:2]: dont do this. dont implement method in your test file

**Review by TwinGan — CHANGES_REQUESTED** [review]
Please think about the task and re-write your test
